tests/regression_tests/weightwindows/test.py

```diff
@@ -92,13 +92,15 @@ def model():
                                 ww_n_lower_bnds,
                                 None,
                                 10.0,
-                                e_bnds)
+                                e_bnds,
+                                max_lower_bound_ratio=1.0)
```

Comment:
I thought this should be greater than one to be valid?

Reply:
Right now it can be greater than or equal. It will only be applied if greater than one, but along the lines of your previous comment perhaps we should require it to be greater than so we don't write one to XML if we aren't going to use it.